src/App/Home/index.js

```diff
@@ -92,55 +58,23 @@ const Home = () => {
 
     const Changelog = () => (
         <section className="dg-changelog-container d-flex flex-column mb-0">
-
-            {/* <a className="dg-next-page-content mt-5 d-none d-lg-flex"
-                onClick={() => window.scrollTo(0, 0)}>
-                <i className="material-icons ">expand_less</i>
-            </a> */}
-
             <div className="container dg-changelog-content my-auto">
                 <div className="row mt-3">
                     <span className="dg-changlelog-title">What’s new in the design guide
                         <div className="dg-changelog-divider my-2"></div>
                     </span>
                 </div>
 
-                <div className="row border-bottom pb-2 mt-4">
-                    <div className="px-0 pb-2">V 5.0.0</div>
-                    <div className="dg-changelog-description px-0">
-                        <span className="h4">Major overhaul of the Developer Portal’s design ♥️</span>
-                        <p className="mt-2">
-                        Happy New Year everyone! We have a brand new look for the Design Guide landing page, a new left side menu updated to suit the branding of Swedbank Pay. We have finally added both “Get started” and “Identity” and re-written all the related pages.
-                        </p>
+                {changeLogs.map(log => (
+                    <div key={log.version} className="row border-bottom pb-2 mt-4">
+                        <div className="px-0 pb-2">Version {log.version}</div>
```

Comment:
Version vs Ver. vs V. etc.
There seems to be some inconsistencies on how to write these, 🤷